developer/packages/taxes/avalara.md

```diff
@@ -0,0 +1,96 @@
+# Avalara
+
+The Avalara Tax plugin provide full integration with the Avalara tax service which provides real-time tax
+computation and Tax return filings.
+
+## First Step: Acquire account information from Avalara
+
+In order to enable the Avalara service in Reaction you must have an API login key, username and password. You can get
+more information and sign up for an account [here](http://developer.avalara.com/)
+
+## Second Step: Configuation
+
+The Avalara configuration screen can be reached from the Tax dashboard panel. Move the slider to the right to enable.
+
+
+![avalara-tax-panel](/assets/avalara-tax-panel.png)
+
+Once enabled you have a number of options to configure
+
+### Configuration Fields
+
+#### Avalara API Login ID
+
+The API Login ID provided to you by Avalara (this is a required field)
+
+#### Username
+
+The Username provided to you by Avalara. This is the same username you use to log into
+the administrator dashboard (this is a required field)
+
+#### Password
+
+The Password provided to you (or created by you) for Avalara. This is the same password
+you would use to log into the administrator dashboard (this is a required field)
+
+#### Company Code
+
+The company code you created in the Avalara administrator panel. (this is a required field)
+
+#### Shipping Tax Code
+
+This is the tax code you want to record Shipping under. By default in Reaction, shipping is not Taxable.
```

Comment:
I think we should add a line about the fact that even though Avalara doesn't list the Shipping Tax Code as required, some functionalities won't work. As such it's labelled as required in the Reaction tax settings.